sphinx-doc/source/downloadable/index.md

```diff
@@ -0,0 +1,7 @@
+# Resources available for download
+
+## Previous versions of the documentation
+
+|              Document                | firmware version | ifm3d version | ifm3d-ros1 version | ifm3d-ros2 version |
+| ------------------------------------ | ---------------- | ------------- | ------------------ | ------------------ |
+| {download}`2022-02-22 <20220222.pdf>`|      13.13       |     0.92.x    |        1.0.0       |       1.0.0        |
```

Comment:
Can we rename it so people can easily identify the PDF when downloaded to their computer: 
Example: ifm3d-com_20220222.pdf

Reply:
Good point. I will merge this and re-generate the pdf in a second step, as I need this merged to generate a complete pdf.